src/locales/cy/translation.json

```diff
@@ -236,6 +236,10 @@
         "deleteInformationLink": "Delete this information from your account",
         "savedOn": "Saved on [date]"
       },
+      "noSavedIdentity": {
+        "heading": "You do not currently have saved identity credentials",
+        "paragraph1": "You will need to do a full identity check next time you use a relevant government service. Or you can do them now to save time later"
```

Comment:
Have we got designs for this content? If not, happy to merge this and ask the UCD people to look at it later

Reply:
Yeah we don't, I think you are right, will ping 'em.